### DISCUSSION TITLE
Add clone button and color tags to items

### DISCUSSION THREAD

**Discussion by mogultekin** [hay-kot/homebox#562] [Ideas]
Hi, thanks for this software. I searched for a clone button while adding items. I am using NetBox and it has a clone button for each object. While adding items in homebox I had to add every item which is identical except the serial number. Also the custom fields I enter are not in the net item. If you can implement clone button next to edit button it would save lots of time when adding identical items like disk drives, tools, etc. 
![image](https://github.com/hay-kot/homebox/assets/81960545/64d7edb2-0340-4728-b60c-562aeba9fa9e)

Also it would be great to add a color selection dropdown like the one in NetBox.
![image](https://github.com/hay-kot/homebox/assets/81960545/b43ea55e-0811-4a00-b807-445b2f820186)

**Comment by larcin78** [comment]
A clone button would be amazing! I just started using this and my first item my monitors I have 3 of and went looking for the clone option and did not find one and came here to suggest it.
